javascript/MaterialXView/source/dropHandling.js

```diff
@@ -198,7 +212,28 @@ async function handleFilesystemEntries(entries) {
         return 0;
     });
 
-    if (debugFileHandling) console.log("all files", allFiles);
+    if (isGLB)
+    {
+        console.log("Load GLB file", allFiles[0]);
+
+        const rootFile = allFiles[0];
+        THREE.Cache.add(rootFile.fullPath, await getBufferFromFile(rootFile));
+
+        if (debugFileHandling) console.log("CACHE", THREE.Cache.files);
+
+        sceneLoadingCallback(rootFile);
+        return;
+    }
+
+    if (!allFiles[0].name.endsWith('mtlx'))
```

Comment:
Add filrewall check if no mtlx file otherwise dropped don't continue as it will cache a bunch of files with no mtlx file.